spec/system/admin/reports_spec.rb

```diff
@@ -52,8 +52,10 @@
       visit admin_report_path(
         report_type: :customers, report_subtype: :mailing_list
       )
+      allow(ENV).to receive(:fetch).and_call_original
```

Comment:
Hehe, I was wondering if I should have committed this separately. This line is only needed when you run the spec on its own. Then there are other `fetch` calls as well. This line allows these other calls. Otherwise Rspec thinks that our expected call should be the only one.

That line should have been in the original commit of the next line. But that was a different PR and I didn't notice that the spec didn't run on its own then.